package/gluon-mesh-babel/Makefile

```diff
@@ -0,0 +1,50 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-mesh-babel
+PKG_VERSION:=1
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+PKG_BUILD_DEPENDS := respondd
+PKG_BUILD_DEPENDS += libbabelhelper
+PKG_BUILD_DEPENDS += libjson-c
+
+include ../gluon.mk
+
+define Package/gluon-mesh-babel
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  TITLE:=Babel mesh
+  DEPENDS:=+gluon-core +babeld +mmfd +libiwinfo +libgluonutil +firewall +libjson-c +libnl-tiny +libubus +libubox +libblobmsg-json +libbabelhelper
+  PROVIDES:=gluon-mesh-provider
+endef
+
+define Build/Prepare
+        mkdir -p $(PKG_BUILD_DIR)
```

Comment:
some lines (like this one) in this Makefile have different indentations than the majority of lines... e.g. the package definition section or line 22 and 35